lib/services/metadata.js

```diff
@@ -59,6 +59,7 @@ function publishPage(uri, publishMeta, user) {
 function createPage(ref, user) {
   const NOW = new Date().toISOString(),
     users = [userOrRobot(user)],
+    site = sitesService.getSiteFromPrefix(ref.substring(0, ref.indexOf('/_pages'))),
```

Comment:
Do you think it would be a good idea to use `getPrefix` method from `clayutils` in here?
```js
site = sitesService.getSiteFromPrefix(getPrefix(ref)),
```